test/parallel/test-http2-perf_hooks.js

```diff
@@ -100,3 +101,8 @@ server.on('listening', common.mustCall(() => {
   }));
 
 }));
+
+process.on('exit', () => {
+  // There shouldn't be any http2 entries left over.
+  assert.strictEqual(performance.getEntries().length, 1);
```

Comment:
what is the one entry that is left in there? Can we match that instead?

Reply:
The nodeTiming entry and yes.